GPU/GLES/StateMapping.cpp

```diff
@@ -783,15 +793,22 @@ void TransformDrawEngine::ApplyDrawState(int prim) {
 	float offsetY = gstate.getOffsetY();
 
 	if (throughmode) {
-		// If the buffer is too large, offset the viewport to the top.
-		renderY += renderHeight - framebufferManager_->GetTargetHeight() * renderHeightFactor;
-
 		// No viewport transform here. Let's experiment with using region.
-		glstate.viewport.set(
-			renderX + displayOffsetX + (0 + regionX1) * renderWidthFactor,
-			renderY + displayOffsetY + (0 - regionY1) * renderHeightFactor,
-			(regionX2 - regionX1) * renderWidthFactor,
-			(regionY2 - regionY1) * renderHeightFactor);
+		if (useBufferedRendering) {
+			// No flip needed
```

Comment:
Do we still need to offset for renderHeight/bufferHeight?

If the bufferHeight is 512, but the viewport is only 272 tall, we need to make sure we're targetting rows 0-271 of the framebuffer.  Are coordinates only flipped within the viewport?

-[Unknown]